conf/messages.cy

```diff
@@ -1,4 +1,3 @@
-base.signOut = Allgofnodi
```

Comment:
I think you removed this by accident instead of the duplicate `back` message?

Reply:
Yep... At least you knew what i was trying to do...:D